net/nimble/host/services/gap/include/services/gap/ble_svc_gap.h

```diff
@@ -37,6 +37,8 @@ struct ble_hs_cfg;
 
 const char *ble_svc_gap_device_name(void);
 int ble_svc_gap_device_name_set(const char *name);
+int ble_svc_gap_device_appearance(void);
+int ble_svc_gap_device_appearance_set(const uint16_t appearance);
```

Comment:
I'd actually make this a syscfg value (BLE_SVC_GAP_APPEARANCE)
the reason for this is that appearance is read-only so it cannot (and should not) be changed in runtime.
getter is ok though.